tough/Cargo.toml

```
@@ -22,7 +22,7 @@ log = "0.4"
 olpc-cjson = { version = "0.1", path = "../olpc-cjson" }
 pem = "3"
 percent-encoding = "2"
-reqwest = { version = "0.11", optional = true, default-features = false, features = ["stream"] }
+reqwest = { version = "0.12", optional = true, default-features = false, features = ["stream"] }
```

```suggestion
reqwest = { version = "0.11", optional = true, default-features = false, features = ["stream"] }
reqwest-hyper-v1 = { package = "reqwest", version = "0.12", optional = true, default-features = false, features = ["stream"] }
```
I wonder if we could do this and separate them by feature. Crates stuck on the old hyper would continue to use `http` and those on the new could use a `http-hyper-v1`. Curious if @bcressey or @cbgbt have thoughts around this as well.